### PR TITLE
Bug fixes and styling

### PR DESCRIPTION
**Pull request by BrassoiCatalin**

### CODE REVIEW

File: SevenSlots/SevenSlots/SevenSlots/SevenSlots.csproj [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
   <ItemGroup>
     <Reference Include="Mono.Android">
-	  <HintPath>C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\IDE\ReferenceAssemblies\Microsoft\Framework\MonoAndroid\v12.0\Mono.Android.dll</HintPath>
+	  <HintPath>C:\Program Files\Microsoft Visual Studio\2022\Professional\Common7\IDE\ReferenceAssemblies\Microsoft\Framework\MonoAndroid\v12.0\Mono.Android.dll</HintPath>
```
Comment (Sorin37) [inline SevenSlots/SevenSlots/SevenSlots/SevenSlots.csproj]: stop pushing this 😠

File: SevenSlots/SevenSlots/SevenSlots/View/CrashView.xaml [hunk 2]
```diff
@@ -34,7 +34,7 @@
                     </CollectionView.Background>
                     <CollectionView.ItemsLayout>
                         <LinearItemsLayout   Orientation="Horizontal"/>
```
Comment (Sorin37) [inline SevenSlots/SevenSlots/SevenSlots/View/CrashView.xaml]: extra space between the tag and the attribute

**Review by Sorin37 — CHANGES_REQUESTED** [review]

File: SevenSlots/SevenSlots/SevenSlots/View/CrashView.xaml (hunk 2 above)
Comment (BrassoiCatalin) [reply SevenSlots/SevenSlots/SevenSlots/View/CrashView.xaml]: I didn't even changed this code, this was from Tinel...

File: SevenSlots/SevenSlots/SevenSlots/ViewModel/BlackjackViewModel.cs [hunk 3]
```diff
@@ -214,6 +214,12 @@ private void Deal()
                 return;
             }
 
+            if(Player.TotalBet > Player.BankRoll)
+            {
+                App.Current.MainPage.DisplayAlert("Alert!", "You Can't Bet More Than You Have!", "OK");
```
Comment (Sorin37) [inline SevenSlots/SevenSlots/SevenSlots/ViewModel/BlackjackViewModel.cs:219]: i suggest that not all the words to start with capital letters

**Review by Sorin37 — APPROVED** [review]

**Pull request merged**